src/frontend/src/tests/lib/utils/token.utils.spec.ts

```diff
@@ -275,3 +280,39 @@ describe('sumUsdBalances', () => {
 		expect(sumUsdBalances([undefined, undefined])).toBeUndefined();
 	});
 });
+
+describe('findTwinToken', () => {
+	it('should return correct twin token', () => {
+		const ckBtcToken = {
+			...mockValidIcCkToken,
+			symbol: BTC_MAINNET_TOKEN.twinTokenSymbol
+		} as IcCkToken;
+
+		const result = findTwinToken({
+			tokens: [...mockTokens, ckBtcToken],
+			tokenToPair: BTC_MAINNET_TOKEN
+		});
+
+		expect(result).toBe(ckBtcToken);
+	});
+
+	it('should return undefined if no twin token found', () => {
+		const result = findTwinToken({
+			tokens: mockTokens,
```

Comment:
`tokens: [...mockTokens, ckBtcToken]` maybe just to show that there is at least a twin token?

Reply:
done!